docs/foundations/icons.md

```diff
@@ -0,0 +1,156 @@
+
+<img width="200" alt="Displaying the use of 2px stroke width throughout icon" src="https://user-images.githubusercontent.com/957314/151464770-67b3f744-8b1a-48e0-91f5-8eb61fd680bf.png">
+
+###### Corners
+
+The corner radius on the outside of the stroke is 1px, while on the inside of the stroke is 0px.
+
+<img width="200" alt="1px corner radius displayed using a circle" src="https://user-images.githubusercontent.com/957314/151464813-5acc3f81-da90-49fe-99a6-d8519bbc4fe1.png">
+
+###### Fill vs. no fill
+
+Currently, SFDesSys icons contain no fill beyond the stroke’s fill.
+
+| <img width="200" alt="Correctly displaying an outlined icon" src="https://user-images.githubusercontent.com/957314/151464853-902b51be-0c6b-4794-b79e-ab711ac32329.png"> | <img width="200" alt="An icon with a fill instead of outlined" src="https://user-images.githubusercontent.com/957314/151464961-e526f296-bb78-45a0-8321-bc5c210792cb.png"> |
+| ----------- | ----------- |
+| Yes      | No       |
+| Icon is drawn with a stroke only | The icon should not be filled with color |
+
+###### Perspective
+
+Make sure to illustrate the icons from the same perspective as the other icons. SFDesSys icons are drawn in a single-point, forward-facing view.
+
+| <img width="200" alt="Icon illustrated from a single-point perspective" src="https://user-images.githubusercontent.com/957314/151465020-1a5c43cc-ba9f-4731-b6ee-b76cfe553a9d.png"> | <img width="200" alt="Icon illustrated in an isometric way" src="https://user-images.githubusercontent.com/957314/151465093-5b4e364d-793b-464b-86e3-ac34465c2cb2.png"> |
+| ----------- | ----------- |
+| Yes      | No       |
+| Icons should be drawn from a single point | Icon is drawn in an isometric style  |
+
+## Usage Guidelines
+
+### When to use
+
+It’s primarily up to the designer when to introduce an icon to their design. The designer should ask if an icon will enhance the experience of the user or create noise and confusion.
+
+### How to use
+
+###### Sizing
+
+The icons are sized at 20px and should generally not be scaled beyond that size.
+
+| <img width="250" alt="icon-size-correct" src="https://user-images.githubusercontent.com/957314/151465258-0f916e68-b94b-4183-9853-2aa8805f3ea6.png"> | <img width="250" alt="icon-size-incorrect" src="https://user-images.githubusercontent.com/957314/151465276-4ee44c10-dbfc-435f-83a0-e0ba1e342f7a.png"> |
+| ----------- | ----------- |
+| Yes      | No       |
+| Icon is implemented at 20px | Icon has been scaled beyond the recommended 20px |
+
+###### Margins & padding
+
+We’ve created all icons 2px padding, and that padding should not be infringed upon by other icons or text.
+
+| <img width="250" alt="icon-margin-correct" src="https://user-images.githubusercontent.com/957314/151465295-84677aab-e5f0-4c66-8935-00b3c4c7d4ac.png"> | <img width="250" alt="icon-margin-incorrect" src="https://user-images.githubusercontent.com/957314/151465312-a47a07b4-e280-4e94-9415-c2a3fb7933cc.png"> |
```

Comment:
yea seems like `width=250` is too large for our page to accommodate. If you replace all instances of `width=250` with `class="w-1/1"` like above we should have at least a temporary fix. I imagine with some of these we'll want to comb through and replace images with the actual markup (e.g. text and svg's) if it's available, though.

Reply:
Yeah, ideally, once the Do/Don't component is made, these would be replaced with live examples